components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/pages/cdmf.page.devices/devices.hbs

```diff
@@ -229,62 +229,84 @@
                 </div>
 
                 <div id="remove-device-modal-content" class="hide">
-                    <div class="content">
-                        <div class="row">
-                            <div class="col-lg-5 col-md-6 col-centered">
-                                <h3>Do you really want to remove this device from your Devices List?</h3>
+                    <div class="modal-header">
+                        <h3 class="pull-left modal-title">
+                            Do you really want to remove this device from your Devices List?
+                        </h3>
+                        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><i class="fw fw-cancel"></i></button>
+                    </div>
+                    <div class="modal-body add-margin-top-2x add-margin-bottom-2x">
 
-                                <div class="buttons">
-                                    <a href="#" id="remove-device-yes-link" class="btn-operations">
-                                        &nbsp;&nbsp;&nbsp;&nbsp;Yes&nbsp;&nbsp;&nbsp;&nbsp;
-                                    </a>
-                                    &nbsp;&nbsp;
-                                    <a href="#" id="remove-device-cancel-link" class="btn-operations">
-                                        &nbsp;&nbsp;&nbsp;&nbsp;Cancel&nbsp;&nbsp;&nbsp;&nbsp;
-                                    </a>
-                                </div>
-                            </div>
+                    </div>
+                    <div class="modal-footer">
+                        <div class="buttons">
+                            <a href="#" id="remove-device-yes-link" class="btn-operations">
+                                &nbsp;&nbsp;&nbsp;&nbsp;Yes&nbsp;&nbsp;&nbsp;&nbsp;
+                            </a>
+                            &nbsp;&nbsp;
```

Comment:
Is this '&nbsp;' needed?
Please remove these and address these using css if required.